src/screens/HomePage/HomePage.tsx

```diff
@@ -102,6 +118,17 @@ export default function HomePage() {
     </>
   );
 
+  const LocationToggle: React.FC = () => {
```

Comment:
sorry-i meant into its own file, similar to the [current Toggle setup](https://github.com/covid-projections/covid-projections/blob/develop/src/screens/HomePage/Toggle/Toggle.tsx) on prod